Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/worker/execution/PCRTestResultScheduler.kt

```diff
@@ -80,6 +76,20 @@ class TestResultScheduler @Inject constructor(
         private const val PCR_TESTRESULT_WORKER_TAG = "DIAGNOSIS_TEST_RESULT_PERIODIC_WORKER"
         private const val PCR_TESTRESULT_WORKER_UNIQUEUNAME = "DiagnosisTestResultBackgroundPeriodicWork"
 
-        private const val TAG = "TestResultScheduler"
+        private const val TAG = "PCRTestResultScheduler"
+
+        /**
+         * Calculate the time for pcr diagnosis key retrieval periodic work
+         *
+         * @return Long
+         *
+         * @see BackgroundConstants.MINUTES_IN_DAY
+         */
+        @VisibleForTesting
+        internal fun getPcrTestResultRetrievalPeriodicWorkTimeInterval(): Long =
+            (
+                BackgroundConstants.MINUTES_IN_DAY /
+                    BackgroundConstants.DIAGNOSIS_TEST_RESULT_RETRIEVAL_TRIES_PER_DAY
+                ).toLong()
```

Comment:
I can't believe klint eats this, :ghost:

Reply:
reformatted it